CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ endif()
 # Only for Mac OS X warnings
 if(APPLE)
   set(CMAKE_MACOSX_RPATH 1)
+  set(CMAKE_FIND_FRAMEWORK NEVER)
```

Comment:
Please don't hard code this here - this can be passed from the command line.
If it's a common setting, please make this a CACHE variable

Reply:
DONE. Changed this to a CACHE variable.